lib/commands/releases.ts

```diff
@@ -64,15 +71,27 @@ export default class ReleasesCmd extends Command {
 
 		const releases = await balena.models.release.getAllByApplication(
 			await getFleetSlug(balena, params.fleet),
-			{ $select: fields },
+			options.json
```

Comment:
@bbugh  Similar to the point above, we can't just select them all without an explicit $select on the fields we want. In here you could just go with keeping your $expand, but also adding a $select: fields above (or $select: jsonFields) if you require aditional fields from the release.